test/httpoison_adapter_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.HttpoisonAdapterTest do
+  use ExUnit.Case, async: true
+
+  alias GlassFactoryApi.HttpoisonAdapter
+
+  setup do
+    bypass = Bypass.open()
+    {:ok, bypass: bypass}
+  end
+
+  describe "get/2" do
+    test "returns a map with the response", %{bypass: bypass} do
+      Bypass.expect_once(bypass, fn conn ->
```

Comment:
@w-sanches I gave up with Mox because it would be tricky to mock `HTTPoison` return, so I've changed it to `ByPass`, could you review it again? 😄